lib/manager/fvm/index.ts

```diff
@@ -0,0 +1,14 @@
+import { ProgrammingLanguage } from '../../constants';
+import { GithubTagsDatasource } from '../../datasource/github-tags';
+import * as npmVersioning from '../../versioning/npm';
+
+export { extractPackageFile } from './extract';
+
+export const language = ProgrammingLanguage.Dart;
+
+export const supportedDatasources = [GithubTagsDatasource.id];
+
+export const defaultConfig = {
+  fileMatch: ['(^|/)\\.fvm/fvm_config.json$'],
+  versioning: npmVersioning.id,
```

Comment:
Flutter uses the same versioning as Dart/Pub. I saw that the Pub manager is using this versioning so I'm using the same one here.

Reply:
Shouldn't we better use semver?

Reply:
Yeah probably, I just thought to align it with Pub.